modules/RockRuby.cmake

```diff
@@ -21,6 +21,10 @@
 # endif()
 #
 # 
+# If installed to a target directory set the following to true to
+# make it project specific
+# set(RUBY_USE_PROJECT_INSTALL_PREFIX TRUE)
```

Comment:
I misunderstood the patch - did not get that it was involving the extensions, not the ruby parts.

I would go for having the same default behaviour in CMake that in the gems ... that is installing the extension under <project_name>/ always.

As far as I can see, that change would not impact anyone, as nobody uses the install except the binary packaging anyways, or ?